packages/catalog-process/src/services/catalogService.ts

```diff
@@ -2237,6 +2290,100 @@ export function catalogServiceBuilder(
 
       return updatedEService;
     },
+    async createEServiceInstanceFromTemplate(
+      templateId: EServiceTemplateId,
+      seed: catalogApi.InstanceEServiceSeed,
+      ctx: WithLogger<AppContext>
+    ): Promise<EService> {
+      ctx.logger.info(`Retrieving EService template ${templateId}`);
+
+      const template = await readModelService.getEServiceTemplateById(
+        templateId
+      );
+
+      if (!template) {
+        throw eServiceTemplateNotFound(templateId);
+      }
+
+      const publishedVersion = template.versions.find(
+        (version) => version.state === eserviceTemplateVersionState.published
+      );
+
+      if (!publishedVersion) {
+        throw eServiceTemplateWithoutPublishedVersion(templateId);
+      }
+
+      const { eService: createdEService, events } = await innerCreateEService(
+        {
+          eServiceSeed: {
+            name: `${template.name} ${seed.instanceId ?? ""}`.trim(),
+            description: template.eserviceDescription,
+            technology: technologyToApiTechnology(template.technology),
+            mode: eServiceModeToApiEServiceMode(template.mode),
+            descriptor: {
+              description: publishedVersion.description,
+              audience: [template.audienceDescription],
```

Comment:
You’re right, the name is definitely misleading.
The audienceDescription refers to the description of who the template is intended for, not the audience itself.
Also, it’s not related to any e-service properties

Reply:
Leave an empty array